Source/Core/Core/PowerPC/PowerPC.h

```diff
@@ -164,7 +164,13 @@ void InjectExternalCPUCore(CPUCoreBase* core);
 void SingleStep();
 void CheckExceptions();
 void CheckExternalExceptions();
-void CheckBreakPoints();
+
+// Return zero if execution should continue, non-zero to stop
+u32 CheckBreakPoints();
```

Comment:
Unless I missed something, this seemed only be used in a boolean context with `CheckBreakPoints() != 0` etc...not to mention you only return 0/1. Any particular reasons against returning a `bool` instead?

Reply:
Is the width of bool guaranteed? I did this because JIT-emitted code interacts with the return value and I wanted the width to be guaranteed and clear.

Reply:
Oh, I actually missed that one since it wasn't a changed line. Suppose its ok then?

Reply:
Then, you should probably put a comment regarding that since it probably applies to ```TriggerBreakPoint``` as well.

Reply:
`TriggerBreakPoint` now returns a bool, and `CheckBreakPoints` implicitly casts to u32 on return. There's a comment above the definition of `CheckBreakPoints` explaining the behaviour.